install/updateVersion.sh

```diff
@@ -125,6 +128,21 @@ function merge_files() {
   ISTIO_INITIALIZER=$DEST/istio-initializer.yaml
   ISTIO_CA_PLUGIN_CERTS=$DEST/istio-ca-plugin-certs.yaml
 
+
+  if [ "$COMPONENT_FILES" = true ]; then
+    echo "generating component files"
+      COMPONENT_DIR=$DEST/components
+      if [ ! -d "$COMPONENT_DIR" ]; then
+        mkdir -p $COMPONENT_DIR
+      fi
+      cat $SRC/istio-ns.yaml.tmpl >> $COMPONENT_DIR/istio-ns.yaml
+      cat $SRC/istio-rbac-beta.yaml.tmpl >> $COMPONENT_DIR/istio-rbac-beta.yaml
+      cat $SRC/istio-mixer.yaml.tmpl >> $COMPONENT_DIR/istio-mixer.yaml
+      cat $SRC/istio-config.yaml.tmpl >> $COMPONENT_DIR/istio-config.yaml
+      cat $SRC/istio-pilot.yaml.tmpl >> $COMPONENT_DIR/istio-pilot.yaml.yaml
+      cat $SRC/istio-ingress.yaml.tmpl >> $COMPONENT_DIR/istio-ingress.yaml
```

Comment:
why can't we ship these files always? or is there an issue?

Reply:
I think it would be good to ship the individual files always and doc that you can have the option of bespoke installation or all-in-one with the istio.yaml.

Reply:
@rshriram I don't see any issue other then the proliferation of installation methods.  Documentation would be needed as @christian-posta pointed out - but we are still working on that for the helm installation method ;)

Reply:
@rshriram I misunderstood your statement.  I think what you propose is not having a -m option and just shipping the output files as they are.  WFM.